moveos/moveos-client/src/client.rs

```diff
@@ -0,0 +1,194 @@
+struct SendTxnRequest {
+    sender_address: String,
+    public_key: Vec<u8>,
+    program: Vec<u8>,
+    sequencer_number: u64,
+    expiration_time: u64,
+    signature: Vec<u8>,
+}
+
+struct SendTxnResponse {
+    // The following three fields represent the ordering information of transactions.
+    // Transaction Hash
+    txn_hash: String,
+    // user's transaction's order: the execution order will follow this order strictly.
+    // e.g. txn with y order must be no later than txn with x order, if x > y
+    order: u128,
+    // The signature of the sequencer on this order information.
+    txn_order_signature: Vec<u8>,
+}
+
+struct GetTxnOrderWitnessRequest {
+    // The following three fields represent the ordering information of transactions.
+    // Transaction Hash
+    txn_hash: String,
+    // user's transaction's order: the execution order will follow this order strictly.
+    // e.g. txn with y order must be no later than txn with x order, if x > y
+    order: u128,
+    // The signature of the sequencer on this order information.
+    txn_order_signature: Vec<u8>,
+}
+
+struct GetTxnOrderWitnessResponse {
+    // Transaction Hash
+    txn_hash: String,
+    // Based on the order information of the transactions provided by the user,
+    // return a verifiable proof of transaction order,
+    // so the user can verify whether the order is valid or not.
+    // And the result could be "nil" means sequencer cannot give the witness, may caused by:
+    // illegal TxnOrder
+    // internal server error
+    // too old txn need to search DA, but bad connection or heavy traffic cause request failed
+    txn_order_witness: Option<Vec<u8>>,
+    error_message: Option<String>,
```

Comment:
after txn_order_witness should have sign of it to ensure it's response from sequencer

Reply:
signature needed